packages/input/src/LionInput.js

```diff
@@ -7,12 +7,21 @@ import { LionField } from '@lion/field';
  * @extends LionField
  */
 export class LionInput extends LionField {
-  get delegations() {
+  static get properties() {
     return {
-      ...super.delegations,
-      target: () => this.inputElement,
-      properties: [...super.delegations.properties, 'readOnly'],
-      attributes: [...super.delegations.attributes, 'readonly'],
```

Comment:
I'm not sure what you gonna do with this, so maybe will revisit after the WIP status changes to done

Reply:
If you know how the Delegate Mixin works, you know that it dynamically creates getters and setters. They override the ones creating by Update Mixin, disallowing us to hook into their observer hooks

Reply:
ok, makes sense then

how about other LionInput based component which you are not changed in this MR? how would they work if `readonly` is not delegated?